client/js/navigation/config.js

```diff
@@ -23,7 +26,30 @@ export const sharedNavigationOptions = navigation => ({
   },
   headerLeft: (
     <TouchableOpacity onPress={() => navigation.toggleDrawer()}>
-      <Icon name="md-menu" size={25} color="white" style={{ margin: 15 }} />
+      <IconEntypo
+        name="menu"
+        size={45}
+        color="#fff"
+        style={{ marginLeft: 10 }}
```

Comment:
Move this to a style object

Reply:
This was part of the navigation feature.